home/templates/about.html

```diff
@@ -11,13 +11,29 @@ <h2>About</h2>
 	<h3>Goal</h3>
 	<p>Our goal is to provide information that will help all students at UMD make informed decisions.</p>
 
-	<h3>Review Criteria</h3>
-	<p>While anyone is free to submit a review, anonymously or otherwise, all reviews are manually verified by us before being displayed on the site. Below are some common reasons reviews get rejected:</p>
+	<h3>Review Writing Tips</h3>
+	<p>If you're not sure what to write, here are some suggestions to get you started:</p>
+	<ul>
+		<li>Did you think the exams or other major assignments were fair?</li>
+		<li>Did you think the professor had reasonable expectations of the students in the course overall?</li>
+		<li>Were you a fan of the professor's teaching style?</li>
+		<li>Was the workload appropriate for the number of credits?</li>
+		<li>Was this course a waste of time or was it beneficial in some way?</li>
+		<li>How much support were you given throughout the semester (office hours, extra resources, etc.)?</li>
+	</ul>
 
+	<p>You're welcome to write your review in anyway you see fit, but here are a few tips to maximize your review's readablility and usefulness:</p>
+	<ul>
+		<li>Clearly structure your thoughts through a pros and cons list, spaced-out paragraphs, or any approach of your choosing. The community will thank you.</li>
+		<li>Keep it short and sweet. Try to avoid writing reviews longer than 4-5 paragraphs to ensure your review is read through in its entirety. Nobody wants to read an essay!</li>
+		<li>Try to avoid excessive use of caps lock. We promise you, whatever you want to write can be written without caps lock!</li>
+	</ul>
+
+	<p>While anyone is free to submit a review, anonymously or otherwise, all reviews are manually verified by us before being displayed on the site. Below are some common reasons reviews get rejected:</p>
 	<ul>
-		<li>Vague negative comments with no elaboration. A review only saying "this professor sucks" would be rejected, for instance.</li>
-		<li>Excessive hostility towards a professor. Negativity or criticism is fine, insults are not.</li>
-		<li>Excessive use of caps lock. Don't make your review a nuisance for others to read, please!</li>
+		<li>Reviews like "really bad", "horrible", or "awesome professor" aren't helpful. Elaborate on why you feel this way about the professor! Remember: users who read your review haven't taken this professor and are relying on YOU to help them make an informed decision.</li>
+		<li>Excessive hostility towards a professor. Negativity or criticism is fine, but insults or accusations are unlikely to lead to an accepted review.</li>
+		<li>Excessive/unnecessary profanity or any discriminatory language. We won't reject your review for a few swear words, but do try to keep it to a minimum. Again, discriminatory language will not be tolerated at all and is grounds for immediate rejection. PlanetTerp is a safe community for everyone and we try our best to keep it this way!</li>
```

Comment:
> Again, discriminatory language will not be tolerated at all and is grounds for immediate rejection. PlanetTerp is a safe community for everyone and we try our best to keep it this way!

feels like excessive emphasis is being placed on this compared to everything else. Can we just leave it at `Excessive/unnecessary profanity or any discriminatory language. We won't reject your review for a few swear words, but do try to keep it to a minimum`? That still explicitly mentions discriminatory language and is in line with the other bullet points.